code/game/machinery/computer/_computer.dm

```diff
@@ -63,9 +67,13 @@
 /obj/machinery/computer/power_change()
 	..()
 	if(stat & NOPOWER)
+		if(soundloop)
+			soundloop.stop()
 		set_light(0)
 	else
 		set_light(brightness_on)
+		if(soundloop)
+			soundloop.start()
```

Comment:
Better to know if for some reason it gets deleted rather than hiding the error.

Reply:
i like to hide my insecurities :^)


but sure I'll go check for any soundloop killing secrets